flatpak/org.heroic.yml

```
@@ -0,0 +1,69 @@
+app-id: org.hgl.heroic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+build-options:
+  build-args:
+    - --share=network
+  append-path: /usr/lib/sdk/node14/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: legendary
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/58/e5/4189abbc61491a5f870961f05692d0f7e6c825ccae861805f186c87834c4/legendary-gl-0.20.22.tar.gz
+        sha256: d05b069ce88113a01f8d40fb79475cf4dfba957ac15a50284951f0486ef225be
+  - name: heroic
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/heroic/flatpak-node/cache
+        npm_config_cache: /run/build/heroic/flatpak-node/node-cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+        npm_config_offline: 'true'
+    subdir: main
+    sources:
+      #- type: archive
+      #  url: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/v2.0.0/heroic-2.0.0.tar.xz
+      #  sha256: 5aa8112359ce892ae3e045a56ec90e8fe7a588139ae58ee52284586afac4c021
+      - type: dir
+        path: ..
+        dest: main
+      - generated-sources.json
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/heroic "$@"
```

```suggestion
          - export TMPDIR=$XDG_RUNTIME_DIR/app/$FLATPAK_ID
          - zypak-wrapper.sh /app/main/heroic "$@"
```
This is required for electron app to avoid some issues in flatpak